src/cr/cube/cube.py

```diff
@@ -160,9 +161,9 @@ def iter_cubes():
                     population=self._population,
                     mask_size=self._min_base,
                 )
-                # --- numeric-mean cubes require inflation to restore their
+                # --- numeric-measures cubes require inflation to restore their
                 # --- rows-dimension, others don't
-                yield cube.inflate() if self._is_numeric_mean else cube
+                yield cube.inflate() if self._is_numeric_measure else cube
```

Comment:
Maybe you right that the name is flaky but IMHO this cannot be reverted back. The reason is that now the inflation needs to be done even for `sum` `ste_dev`  or whatever other numeric measure (AKA Quantity of Interests measures) we will have. So `is_numeric_mean`  is meaningless too 🤷🏽‍♂️

Reply:
Okay, I'll buy that, and actually I think `._is_numeric_measure` is quite a good name if that's what we're using this for. But you're going to need to update the docstring for `._is_numeric_measure` because it still says `True when CubeSet is special-case "numeric-mean" case requiring inflation."

Also, the logic in there seems odd to me, is it true that whenever the rows-dimension cube measure is sum or std_dev that it has 0 dimensions (is a Nub)? If so, then a comment in there would be good. Since there were no changes to that property other than the name I assumed nothing had changed, but it sounds like that same logic now has broader applicability.